jdbi/src/main/java/org/killbill/commons/jdbi/guice/DataSourceProvider.java

```diff
@@ -230,6 +243,57 @@ DataSource buildDataSource() {
         }
     }
 
+    private class UnpooledDataSourceBuilder {
+
+        DataSource buildDataSource() {
+            String dataSourceClassName = config.getDataSourceClassName();
+            if (Strings.isNullOrEmpty(dataSourceClassName)) {
+                throw new IllegalArgumentException("dataSourceClassName was not specified in the configuration and could not be inferred");
+            }
+
+            if (DatabaseType.MYSQL.equals(databaseType)) {
+                Reference ref = new Reference(dataSourceClassName);
+                ref.add(new StringRefAddr("user", config.getUsername()));
+                ref.add(new StringRefAddr("password", config.getPassword()));
+                ref.add(new StringRefAddr("explicitUrl", "true"));
+                ref.add(new StringRefAddr("url", config.getJdbcUrl()));
+                try {
+                    return (DataSource) new MysqlDataSourceFactory().getObjectInstance(ref, null, null, null);
```

Comment:
@rhanneken Any update? We 'd like to release `0.18.3` (we have a few bug fixes for issues you guys reported) and we would like to know if we should include your change for DB pooling?